frb_codegen/src/config/opts.rs

```diff
@@ -29,17 +43,220 @@ pub struct Opts {
     pub bridge_in_method: bool,
     pub extra_headers: String,
     pub dart3: bool,
+    pub shared: ShareMode, // it is true if this Opts instance is for auto-generated shared API block. Otherwise, it is false,
```

Comment:
Maybe it should be renamed to `shared_mode` now?

Reply:
Yes, renaming those `shared` field or  params with `mode` does make it more readable.  I rename it with `share_mode` in [19c727c](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1145/commits/19c727c2efe83e8a08843d6b703c217155d00bf6).